cmd/exporters/influxdb/influxdb_test.go

```diff
@@ -6,27 +6,36 @@ package influxdb
 import (
 	"goharvest2/cmd/poller/exporter"
 	"goharvest2/cmd/poller/options"
+	"goharvest2/pkg/conf"
 	"goharvest2/pkg/matrix"
-	"goharvest2/pkg/tree/node"
 	"testing"
 )
 
 // test that the addr (and port) parameters
 // are handled properly to construct server URL
 func TestAddrParameter(t *testing.T) {
-
 	expectedURL := "http://localhost:8086/api/v2/write?org=netapp&bucket=harvest&precision=s"
 
 	opts := &options.Options{}
 	opts.Debug = true
+	var params map[string]conf.Exporter
```

Comment:
minor but somewhat consistent throughout this PR - it would be better to call these exporters - it made sense to call it params earlier with the node code, but it's clearer to call it exporters now

Reply:
done